examples/starter/.storybook/preview.ts

```diff
@@ -1,6 +1,4 @@
-import type { Preview } from '@nuxtjs/storybook'
```

Comment:
Why was that changed?

Reply:
I was normalizing the configs while chasing that render bug. I will revert these.